src/3d/symbols/qgspoint3dsymbol.cpp

```diff
@@ -20,6 +20,7 @@
 
 
 QgsPoint3DSymbol::QgsPoint3DSymbol()
+  : mAltClamping( AltClampRelative )
```

Comment:
Best to use modern way of member initialization in the class definition (see e.g. how mShape is initialized)

Reply:
@wonder-sk , I was copying the style found in qgsline3dsymbol and qgspolygon3dsymbol. Should I stick to that, or update all three to the modern way?

Reply:
I'm easy - either way works for me :-)